src/ResourceManager/Common/Commands.ResourceManager.Common/Commands.ResourceManager.Common.csproj

```diff
@@ -92,6 +92,10 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\..\packages\Microsoft.WindowsAzure.Management.4.1.1\lib\net40\Microsoft.WindowsAzure.Management.dll</HintPath>
     </Reference>
+    <Reference Include="Moq, Version=4.2.1510.2205, Culture=neutral, PublicKeyToken=69f491c39445e920, processorArchitecture=MSIL">
```

Comment:
@rnikhilkumar Do not add this reference here - this adds a dependency to every ARM cmdlet that is not needed.